pkg/psmdb/backup/backup.go

```diff
@@ -28,11 +29,17 @@ func NewBackupJob(name string) Job {
 	}
 }
 
-func NewRestoreJob(name string) Job {
-	return Job{
-		Name: name,
+func NewRestoreJob(cr *api.PerconaServerMongoDBRestore) (j Job) {
```

Comment:
Please avoid named return params
It's too easy to do mistake with it. And we do not use it in our code, so let's follow code style

Reply:
removed in 91ee176